contracts/access/Ownable2Step.sol

```diff
@@ -0,0 +1,64 @@
+// SPDX-License-Identifier: MIT
+// OpenZeppelin Contracts (last updated v4.7.0) (access/Ownable.sol)
+
+pragma solidity ^0.8.0;
+
+import "./Ownable.sol";
+
+/**
+ * @dev Contract module which provides access control mechanism, where
+ * there is an account (an owner) that can be granted exclusive access to
+ * specific functions.
+ *
+ * By default, the owner account will be the one that deploys the contract. This
+ * can later be changed with {transferOwnership} and {acceptOwnership}.
+ *
+ * This module is used through inheritance. It will make available all functions
+ * from parent (Ownable).
+ */
+abstract contract Ownable2Step is Ownable {
+    address private _pendingOwner;
+
+    event OwnershipTransferStarted(address indexed newOwner);
+
+    /**
+     * @dev Returns the address of the pending owner.
+     */
+    function pendingOwner() public view virtual returns (address) {
+        return _pendingOwner;
+    }
+
+    /**
+     * @dev Starts the ownership transfer of the contract to a new account
+     * Can only be called by the current owner.
+     */
+    function transferOwnership(address newOwner) public virtual override onlyOwner {
+        _pendingOwner = newOwner;
+        emit OwnershipTransferStarted(newOwner);
+    }
+
+    /**
+     * @dev Make the direct ownership transfer available as a fallback when the receiver cannot accept.
+     */
+    function forceTransferOwnership(address newOwner) public virtual {
```

Comment:
the key here is to know what will happen to the `renounceOwnership` function?
**IF** there is chance that will be removed we need a way to still allow it ...
... or we could cross that bridge when we get to it

Reply:
> This one should be removed IMO. cc @Amxx I think it was added based on your suggestion?

Yes, I think its nice to keep it available as an unsafe alternative.
We are not enforcing any delay in transfer → accept ... so I don't think it breaks any assumption.

Reply:
Yes, there should definitely be a test for `renounceOwnership` anyway. The way that the pending owner is cleared on renounce is kind of implicit so it should be tested.

Reply:
If we're keeping both transfer methods, I would consider keeping `transferOwnership` with the basic behavior, and instead add an additional function "start ownership transfer".

Reply:
That was also my original idea, but community contributors said (in the issue) that it would defeat the point since by default people would keep using the same behavior.

The idea of that extension was to make the 2 step transfer the default workflow, and require users to move out of their way to user the "unsafe" version.

Reply:
....after further reflection, I'm leaning towards removing the forceTransferOwnership, the renounceOwnership might never be removed and  possibly a better solution might be available then

Reply:
I guess the internal `_transferOwnership` stays available is someone want to use it / expose it

Reply:
Perfect, I think we all on agreement ... I will remove the `forceTransferOwnership` and add a test on `renounceOwnership`

Reply:
done